packages/amplify-cli/src/__tests__/commands/uninstall.pkg.test.ts

```diff
@@ -60,6 +60,18 @@ describe('uninstall packaged CLI on mac / linux', () => {
       value: originalPlatform,
     });
   });
+
+  expect.addSnapshotSerializer({
+    test(val) {
+      return typeof val === 'string';
+    },
+    print(val) {
+      if (typeof val === 'string')
```

Comment:
You should be able to remove this `if` statement.

Reply:
It says that val has "undefined" type and this replace action won't be permitted, thus the if statement
yes, it should have gotten tested in test()

Reply:
That is a TypeScript issue. You can try changing the function signature to `print(val: string)` here. If that doesn't work, you can force it by using `val as string` in the function body.